git-gud.1

```diff
@@ -0,0 +1,68 @@
+.TH git-gud 1 "19 Feb 2020" "version 0.0"
```

Comment:
Git is written `GIT(1)` in its man page, maybe we should make this all caps too?

Reply:
Ope, I must have accidentally held Shift while CapsLock was on. I will fix this.

Reply:
Fixed